src/ui/utils/telemetry.ts

```diff
@@ -112,11 +112,26 @@ export function setTelemetryContext({ id, email, internal }: TelemetryUser) {
   }
 }
 
-export function sendTelemetryEvent(event: string, tags: any = {}) {
+const sentEvents = new Set<string>();
+
+type TelemetryOptions = {
+  /** It can be used to avoid sending repeated events. It's always combined with `event` for that purpose */
+  id?: string;
```

Comment:
I think "id" might be a bit too ambiguous. I wonder if something like `distinctOn` might read more intuitively at the call sight?

Reply:
i like that 👍